WordPress/Jetpack/Universal Links/JetpackRoute+Action.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+extension JetpackRoute: NavigationAction {
+
+    func perform(_ values: [String: String], source: UIViewController?, router: LinkRouter) {
+        guard !AccountHelper.isLoggedIn,
+              let windowManager = WordPressAppDelegate.shared?.windowManager as? JetpackWindowManager
+        else {
+            return
+        }
+        do {
+            let migrator = DataMigrator()
+            try migrator.importData()
```

Comment:
For some reason the completion handler wasn't called. Maybe because the `migrator` instance gets deallocated before the completion is called? 

Since there is nothing `async` about the `importData` method, I think it's fine to change the method signature to:

```
func importData() throws -> Void
```